sockeye/inference.py

```diff
@@ -610,7 +621,7 @@ def _concat_translations(translations: List[Translation],
         if idx == len(translations) - 1:
             target_ids.extend(translation.target_ids)
         else:
-            if translation.target_ids[-1] in stop_ids:
+            if translation.target_ids[-1][0] in stop_ids:
```

Comment:
What about target factors, wouldn't we need to concatenate them as well?

Reply:
`target_ids` is multi-dimensional, i.e. a nested list. So we check on the primary factor whether its a stop id, but we extend correctly.

Reply:
ohh ok, got it, makes sense!